cmd/mungedocs/util.go

```diff
@@ -141,6 +134,13 @@ type fileBlock struct {
 
 type fileBlocks []fileBlock
 
+var (
+	// Finds all preformatted block start/stops.
+	preformatRE    = regexp.MustCompile("^\\s*```")
+	notPreformatRE = regexp.MustCompile("^\\s*```.*```")
+	preformatEndRE = regexp.MustCompile("^\\s*```")
```

Comment:
Might as well just reuse preformatRE if they are the same. I had it separate to parse some (incorrect?) md files we had, but they've been fixed, I guess.